snapshotEngine/getLatestSnapshotMetadata.py

```diff
@@ -46,7 +46,10 @@
         # Parses date from tezos build of each artifact, compares to last date, updates if older, otherwise its newer
         for snapshot in snapshots:
             matches=datefinder.find_dates(snapshot['tezos_version'])
-            tezos_build_datetime=list(matches)[0]
+            try:
+                tezos_build_datetime=list(matches)[0]
+            except IndexError:
+                continue
             if tezos_build_datetime < last_tezos_build_datetime:
```

Comment:
I Believe either of these should work. But I would test.

```suggestion
            tezos_build_datetime=list(matches)[0] if len(matches) else None
            if tezos_build_datetime and tezos_build_datetime < last_tezos_build_datetime:
```
-----
```suggestion
            try:
                tezos_build_datetime=list(matches)[0]
            except IndexError:
                tezos_build_datetime = None
            if tezos_build_datetime and tezos_build_datetime < last_tezos_build_datetime:
```